src/pages/programs/program-assessment-submission/program-assessment-submission.html

```diff
@@ -1,8 +1,5 @@
 <ion-header>
```

Comment:
This page is changed every where for submissionId and submissionStatus to _id and status